internal/pkg/warewulfd/provision.go

```diff
@@ -92,42 +93,44 @@ func ProvisionSend(w http.ResponseWriter, req *http.Request) {
 		}
 
 	} else if rinfo.stage == "ipxe" {
-		stage_file = path.Join(conf.Paths.Sysconfdir, "warewulf/ipxe/"+node.Ipxe.Get()+".ipxe")
+		stage_file = path.Join(conf.Paths.Sysconfdir, "warewulf/ipxe/"+remoteNode.Ipxe+".ipxe")
 		tmpl_data = templateVars{
-			Id:             node.Id.Get(),
-			Cluster:        node.ClusterName.Get(),
-			Fqdn:           node.Id.Get(),
+			Id:             remoteNode.Id(),
+			Cluster:        remoteNode.ClusterName,
+			Fqdn:           remoteNode.Id(),
 			Ipaddr:         conf.Ipaddr,
 			Port:           strconv.Itoa(conf.Warewulf.Port),
-			Hostname:       node.Id.Get(),
+			Hostname:       remoteNode.Id(),
 			Hwaddr:         rinfo.hwaddr,
-			ContainerName:  node.ContainerName.Get(),
-			KernelArgs:     node.Kernel.Args.Get(),
-			KernelOverride: node.Kernel.Override.Get()}
+			ContainerName:  remoteNode.ContainerName,
+			KernelArgs:     remoteNode.Kernel.Args,
+			KernelOverride: remoteNode.Kernel.Override}
+
 	} else if rinfo.stage == "kernel" {
-		if node.Kernel.Override.Defined() {
-			stage_file = kernel.KernelImage(node.Kernel.Override.Get())
-		} else if node.ContainerName.Defined() {
-			stage_file, _, err = kernel.FindKernel(container.RootFsDir(node.ContainerName.Get()))
-			if err != nil {
-				wwlog.Error("No kernel found for container %s: %s", node.ContainerName.Get(), err)
+		if remoteNode.Kernel.Override != "" {
+			stage_file = kernel.KernelImage(remoteNode.Kernel.Override)
+		} else if remoteNode.ContainerName != "" {
+			stage_file = container.KernelFind(remoteNode.ContainerName)
```

Comment:
looks like you need to rebase this branch, the function could not be found during make.